README.md

```diff
@@ -125,6 +128,39 @@ jobs:
         file_glob: true
 ```
 
+Example with `repo_name`:
```

Comment:
This should explain a bit more. Something like

    Example for creating a release in a foreign repository using `repo_name`: